### PR TITLE
Add mozorg/about/manifesto.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9829

### CODE REVIEW

File: en/mozorg/about/manifesto.ftl [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+manifesto-details-the-mozilla-project-community = The { -brand-name-mozilla } project uses a community-based approach to create world-class open source software and to develop new types of collaborative activities. We create communities of people involved in making the internet experience better for all of us.
+manifesto-details-as-a-result-of = As a result of these efforts, we have distilled a set of principles that we believe are critical for the internet to continue to benefit the public good as well as commercial aspects of life. We set out these principles below.
+manifesto-details-the-goals-for = The goals for the Manifesto are to:
+manifesto-details-articulate-a-vision = articulate a vision for the internet that { -brand-name-mozilla } participants want the { -brand-name-mozilla } Foundation to pursue;
```
Comment (peiying2) [inline en/mozorg/about/manifesto.ftl:96]: `the { -brand-name-mozilla } Foundation` -> `the { -brand-name-mozilla-foundation }`

**Review by peiying2 — CHANGES_REQUESTED** [review]
- A few related to brand name Mozilla Foundation
- Lightbeam and Maker Party are rarely used. Should we include them in brands.ftl? This is mostly to keep them in English, especially if we ask the vendor to translate some of the languages. It is an easier fix if there is a problem. But I am fine if we don't want to include the these two entries.

**Comment by alexgibson** [comment]
@peiying2 thanks for the review, updated.

**Review by peiying2 — APPROVED** [review]
Thanks for the fixes. LGTM!

**Pull request merged**